ccloud/ccloud-generate-cp-configs.sh

```diff
@@ -98,3 +133,259 @@ while read -r line
+    { "sasl.username", "$CLOUD_KEY" },
+    { "sasl.password", "$CLOUD_SECRET" },
+    { “plugin.library.paths”, “monitoring-interceptor”},
+    // .... additional configuration settings
+};
+
+var consumerConfig = new Dictionary<string, object>
+{
+    { "bootstrap.servers", "$BOOTSTRAP_SERVERS" },
+    { "broker.version.fallback", "0.10.0.0" },
+    { "api.version.fallback.ms", 0 },
+    { "sasl.mechanisms", "PLAIN" },
+    { "security.protocol", "SASL_SSL" },
+    { "ssl.ca.location", "/usr/local/etc/openssl/cert.pem" }, // linux, osx
+    // { "ssl.ca.location", "c:\\path\\to\\cacert.pem" },     // windows
+    { "sasl.username", "$CLOUD_KEY" },
+    { "sasl.password", "$CLOUD_SECRET" },
+    { “plugin.library.paths”, “monitoring-interceptor”},
+    // .... additional configuration settings
+};
+EOF
+
+# Go
+GO_CONFIG=$DEST/go.delta
+echo "$GO_CONFIG"
+
+cat <<EOF >> $GO_CONFIG
+import (
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+producer, err := kafka.NewProducer(&kafka.ConfigMap{
+	         "bootstrap.servers": "$BOOTSTRAP_SERVERS",
+	         "broker.version.fallback": "0.10.0.0",
+	         "api.version.fallback.ms": 0,
+	         "sasl.mechanisms": "PLAIN",
+	         "security.protocol": "SASL_SSL",
+	         "sasl.username": "$CLOUD_KEY",
+	         "sasl.password": "$CLOUD_SECRET",
+                 "plugin.library.paths": "monitoring-interceptor",
+                 // .... additional configuration settings
+                 })
+
+consumer, err := kafka.NewConsumer(&kafka.ConfigMap{
+		 "bootstrap.servers": "$BOOTSTRAP_SERVERS",
+		 "broker.version.fallback": "0.10.0.0",
+		 "api.version.fallback.ms": 0,
+		 "sasl.mechanisms": "PLAIN",
+		 "security.protocol": "SASL_SSL",
+		 "sasl.username": "$CLOUD_KEY",
+		 "sasl.password": "$CLOUD_SECRET",
+		 "session.timeout.ms": 6000,
+                 "plugin.library.paths": "monitoring-interceptor",
+                 // .... additional configuration settings
+                 })
+EOF
+
+# Node.js
+NODE_CONFIG=$DEST/node.delta
+echo "$NODE_CONFIG"
+
+cat <<EOF >> $NODE_CONFIG
+var Kafka = require('node-rdkafka');
+
+var producer = new Kafka.Producer({
+    'metadata.broker.list': '$BOOTSTRAP_SERVERS',
+    'sasl.mechanisms': 'PLAIN',
+    'security.protocol': 'SASL_SSL',
+    'sasl.username': '$CLOUD_KEY',
+    'sasl.password': '$CLOUD_SECRET',
+    'plugin.library.paths': 'monitoring-interceptor',
+    // .... additional configuration settings
+  });
+
+var consumer = Kafka.KafkaConsumer.createReadStream({
+    'metadata.broker.list': '$BOOTSTRAP_SERVERS',
+    'sasl.mechanisms': 'PLAIN',
+    'security.protocol': 'SASL_SSL',
+    'sasl.username': '$CLOUD_KEY',
+    'sasl.password': '$CLOUD_SECRET',
+    'plugin.library.paths': 'monitoring-interceptor',
+    // .... additional configuration settings
+  }, {}, {
+    topics: '<topic name>',
+    waitInterval: 0,
+    objectMode: false
+});
+EOF
+
+# C++
+CPP_CONFIG=$DEST/cpp.delta
+echo "$CPP_CONFIG"
+
```

Comment:
cppkafka is a community-maintained wrapper for librdkafka, is this really what we want to use?
We really don't know much about its quality, stability or governance.

I'd suggest using the librdkafka API directly, which is more verbose but at least it is something we support.

Reply:
@edenhill , this was unintentional on my part, total newbie mistake.  I will update the PR accordingly.  Thank you for the feedback.

FYI how I made the mistake: I saw `cppkafka` under https://github.com/edenhill/librdkafka/#language-bindings and went directly there.  I didn't see the librdkafka API examples for cpp until now.

Reply:
@edenhill : I have updated the code to use `librdkafka` instead of `cppkafka`.  All feedback welcome.